File: app/src/main/java/com/rtukpe/fixtures/ui/competition/teams/team/TeamMvpView.java
package com.rtukpe.fixtures.ui.competition.teams.team;

import com.rtukpe.fixtures.data.model.Player;
import com.rtukpe.fixtures.ui.base.DialogMvpView;

import java.util.ArrayList;

public interface TeamMvpView extends DialogMvpView {

    void updatePlayers(ArrayList<Player> players);
}

File: app/src/main/java/com/rtukpe/fixtures/ui/competition/teams/team/TeamMvpView.kt
package com.rtukpe.fixtures.ui.competition.teams.team

import com.rtukpe.fixtures.data.model.Player
import com.rtukpe.fixtures.ui.base.DialogMvpView
import java.util.*

interface TeamMvpView : DialogMvpView {

    fun updatePlayers(players: ArrayList<Player>)
}
